cockatrice/src/game/filters/filter_string.cpp

```diff
@@ -361,11 +361,11 @@ FilterString::FilterString(const QString &expr)
     }
 
     search.log = [&](size_t ln, size_t col, const std::string &msg) {
-        _error = QString("%1:%2: %3").arg(ln).arg(col).arg(QString::fromStdString(msg));
+        _error = QString("line %1, col %2: %3").arg(ln).arg(col).arg(QString::fromStdString(msg));
```

Comment:
if we're going for user friendlyness, line will often be 0, so hide it from the user in that case, col for column is not clear to most users, consider variable width fonts which don't align by column are used by default in this box, instead we could try "character" or "key" or "symbol" or "glyph"

Reply:
just "position"?

Reply:
hey, what I meant with "in that case" is to make it not be there in case it is 0, I don't know in what cases it might not be 0 but it could potentially be very confusing

Reply:
oops... Well, free feel to open up a PR to fix it